packages/dataiku/azure/ansible/roles/dataiku-upgrade/tasks/main.yml

```diff
@@ -80,36 +80,41 @@
     dest: "{{dss_install_directory}}"
     creates: "{{dss_home_directory}}"
     remote_src: yes
-  tags: [dss-upgrade-production, dss-upgrade-training]
+  tags: [ dss-upgrade-production, dss-upgrade-training, dss-upgrade-automation ]
 
 - name: Run DSS upgrade
   become_user: "{{dss_system_user}}"
   shell: yes Y | {{dss_home_directory}}/installer.sh -d {{dss_data_directory}} -u
-  tags: [dss-upgrade-production, dss-upgrade-training]
+  tags: [ dss-upgrade-production, dss-upgrade-training, dss-upgrade-automation ]
+
+- name: Run DSS upgrade
+  become_user: "{{dss_system_user}}"
+  shell: yes Y | {{dss_home_directory}}/installer.sh -d {{dss_data_directory}} -t automation -u
+  tags: [dss-upgrade-automation ]
 
```

Comment:
make the name Run DSS upgrade automation

Reply:
Changed the name